partner_academic_title/models/res_partner.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2015-2017 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    def _get_separator(self):
+        return ', '
+
+    @api.depends('academic_title_ids', 'academic_title_ids.sequence')
+    @api.one
+    def _compute_academic_title_display(self):
+        display_title = ""
+        separator = self._get_separator()
+        title_ids = self.academic_title_ids.sorted(lambda r: r.sequence)
+        for title in title_ids:
```

Comment:
More pythonic (and readable) to do something like separator.join([title.name for title in title_ids])